cordova-lib/spec-cordova/helpers.js

```diff
@@ -43,17 +54,101 @@ if (host.match(/win/)) {
 }
 */
 
-// Just use Android everywhere; we're mocking out any calls to the `android` binary.
-module.exports.testPlatform = 'android';
+module.exports.setEngineSpec = function (appPath, spec) {
+    appPath = getConfigPath(appPath);
+    var parser = new ConfigParser(appPath);
+
+    parser.removeEngine(module.exports.testPlatform);
+    parser.addEngine(module.exports.testPlatform, spec);
+    parser.write();
+};
+
+module.exports.getEngineSpec = function (appPath) {
+    appPath = getConfigPath(appPath);
+    var parser = new ConfigParser(appPath),
+        engines = parser.getEngines();
+
+    for (var i = 0; i < engines.length; i++) {
+        if (engines[i].name === module.exports.testPlatform) {
+            return engines[i].spec;
+        }
+    }
+    return null;
+};
+
+module.exports.removeEngine = function (appPath) {
```

Comment:
please, pass in the engine to be removed as a separate parameter. I think that makes the intent of the function clearer.
Hate global vars, :)